### PR TITLE
Add peaq parachain logo (polkadot)

### PR DESCRIPTION
**Pull request by lavish0000**
This PR adds new parachain for polkadot

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import type { EndpointOption } from './types';
 
 import { POLKADOT_GENESIS } from '../api/constants';
-import { chainsAcalaSVG, chainsBitgreenPNG, chainsComposableFinancePNG, chainsEquilibriumSVG, chainsFrequencySVG, chainsGeminisPNG, chainsOakPNG, chainsOrigintrailPNG, chainsPendulumSVG, chainsPolkadotCircleSVG, chainsSnakenetSVG, chainsTotemSVG, chainsWatrPNG } from '../ui/logos/chains';
+import { chainsAcalaSVG, chainsBitgreenPNG, chainsComposableFinancePNG, chainsEquilibriumSVG, chainsFrequencySVG, chainsGeminisPNG, chainsOakPNG, chainsOrigintrailPNG, chainsPeaqPNG, chainsPendulumSVG, chainsPolkadotCircleSVG, chainsSnakenetSVG, chainsTotemSVG, chainsWatrPNG } from '../ui/logos/chains';
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayPolkadot.ts:7]: The generated file doesn't seem to be checked in? (See https://github.com/polkadot-js/apps/actions/runs/4294366142/jobs/7484496550#step:4:527)

Just re-run the [yarn build:images](https://github.com/polkadot-js/apps/tree/master/packages/apps-config#endpoints) command and add in the resulting file if it passes locally.

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts [hunk 2]
```diff
@@ -499,6 +499,16 @@ export const prodParasPolkadot: EndpointOption[] = [
       logo: nodesParallelSVG
     }
   },
+  {
+    homepage: 'https://peaq.network/',
+    info: 'peaq',
+    paraId: 3338,
+    text: 'Peaq',
+    providers: {},
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayPolkadot.ts:507]:
```suggestion
    providers: {},
    text: 'Peaq',
```

This should fix the linting issues.

**Comment by lavish0000** [comment]
Hey @jacogr , Thanks for reviewing. Please let me know if there is anything else from my side.

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**